conans/search/search.py

```diff
@@ -103,6 +103,8 @@ def search_packages(paths, reference, query):
                            settings: {os: Windows}}}
     param conan_ref: ConanFileReference object
     """
+    if not search_recipes(paths, reference):
```

Comment:
Dangerous, this search by pattern, why is this necessary?

Reply:
This is done to keep consistency with a remote `search_packages()`. If you search packages of a recipe that does not exist in the remote, the current behavior raises "Recipe not found". This was not happening with a local cache `search_packages()`

Reply:
And what about check directly if the recipe folder is present?